### PR TITLE
chore(operator, yaml): add openebs operator 2.0.0

### PR DESCRIPTION
**Pull request by akhilerm**
- add openebs operator 2.0.0-RC1 yaml
Signed-off-by: Akhil Mohan <akhil.mohan@mayadata.io>

### CODE REVIEW

File: 2.0.0/zfs-operator-centos7-0.9.1-RC1.yaml [hunk 1]
```diff
@@ -0,0 +1,1728 @@
...
+            - "--leader-election-type=leases"
+            - "--extra-create-metadata=true"
+          env:
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/lib/csi/sockets/pluginproxy/
+        - name: openebs-zfs-plugin
+          image: quay.io/openebs/zfs-driver:0.9.0
```
Comment (pawanpraka1) [inline 2.0.0/zfs-operator-centos7-0.9.1-RC1.yaml:1414]: version needs to change to 0.9.1-RC1.

File: 2.0.0/zfs-operator-centos7-0.9.1-RC1.yaml [hunk 2]
```diff
@@ -0,0 +1,1728 @@
...
+      role: openebs-zfs
+  serviceName: "openebs-zfs"
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: openebs-zfs-controller
+        role: openebs-zfs
+        openebs.io/component-name: openebs-zfs-controller
+        openebs.io/version: 0.9.0
```
Comment (pawanpraka1) [inline 2.0.0/zfs-operator-centos7-0.9.1-RC1.yaml:1350]: version 0.9.1-RC1

Comment (pawanpraka1) [reply 2.0.0/zfs-operator-centos7-0.9.1-RC1.yaml:1350]: in other yamls also this needs to be changed.

Comment (akhilerm) [reply 2.0.0/zfs-operator-centos7-0.9.1-RC1.yaml]: Done.

File: 2.0.0/zfs-operator-centos7-0.9.1-RC1.yaml (hunk 1 above)
Comment (akhilerm) [reply 2.0.0/zfs-operator-centos7-0.9.1-RC1.yaml]: Done.

File: 2.0.0/zfs-operator-centos7-0.9.1-RC1.yaml (hunk 2 above)
Comment (pawanpraka1) [reply 2.0.0/zfs-operator-centos7-0.9.1-RC1.yaml]: ubuntu and centos 8yamls  still have 0.9.0.

File: 2.0.0/openebs-operator-2.0.0-RC2.yaml [hunk 3]
```diff
@@ -0,0 +1,777 @@
...
+        # If not present, "maya-apiserver-service" will be used for lookup.
+        # This is supported for openebs provisioner version 0.5.3-RC1 onwards
+        #- name: OPENEBS_MAYA_SERVICE_NAME
+        #  value: "maya-apiserver-apiservice"
+         # Process name used for matching is limited to the 15 characters
+         # present in the pgrep output.
+         # So fullname can't be used here with pgrep (>15 chars).A regular expression
+         # that matches the entire command name has to specified.
+         # Anchor `^` : matches any string that starts with `openebs-provis`
+         # `.*`: matches any string that has `openebs-provis` followed by zero or more char
```
Comment (mittachaitu) [inline 2.0.0/openebs-operator-2.0.0-RC2.yaml:311]: Add `LEADER_ELECTION_ENABLED` env in openebs-provisioner also. The same comment applies to other openebs-operator YAMLs also.

Comment (akhilerm) [reply 2.0.0/openebs-operator-2.0.0-RC2.yaml:315]: Done @mittachaitu . Added from RC2 yamls onwards since we introduced it then.

**Pull request merged**